locales/it/index.yml

```diff
@@ -1579,7 +1579,7 @@ cardComponent:
   setFavorite: Imposta come preferita
   unsetFavorite: Rimuovi dai preferiti
   deleteTitle: Vuoi eliminare questo metodo?
-  deleteMsg: Questo metodo di pagamento verrà rimosso. Eventuali funzioni collegate, come il Cashback, verranno disattivate entro 24h.
+  deleteMsg: Questo metodo di pagamento verrà rimosso. Eventuali funzioni collegate verranno disattivate entro 24h.
```

Comment:
@AliceAzzolini Qui sarei più esplicito, qualcosa tipo: "La disattivazione di questo metodo su eventuali iniziative in corso avverrà entro 24h"